code/restful-api/restful-api/Models/RedesSociais.cs

```diff
@@ -0,0 +1,5 @@
+using System;
+public class RedesSociais
```

Comment:
Why do we have `IRedesSociais` and this?

Reply:
Retirada a classe RedeSociais

Reply:
🆗